updatehub/tests/successful_integration_test.rs

```diff
@@ -11,7 +11,7 @@ use common::{
 pub mod common;
 
 #[test]
-fn correct_config_no_update_no_polling() {
```

Comment:
The redundancy here is actually useful. It helps us run tests by partially matching the test names. This commit should be reworked in order to rename all integration tests to mach the following use cases:

- Match all correct usage logging tests;
- Match all failing usage logging tests;
- Match all with api calls;
- Match all tests without api calls;

Either test names or submodules can be use to make this matching possibles.